### DISCUSSION TITLE
Western Digital hard drives and sleep question

### DISCUSSION THREAD

**Discussion by chivesz** [Chia-Network/chia-blockchain#3011] [Q&A] [answered]
Hello,

I purchased two western digital element external hdd's because they were recommended in the documentation.

They go to sleep every minute, and when they go to wake up it takes them about 20 seconds.

If I have a winning plot, I believe they have 30 seconds to respond, so this is kind of concerning.

From what I've read, they are set like this on a firmware level and no settings/windows settings can change this. The only solution I've found would be installing a program called "keep alive" that runs every so often, and writes a small file to the drive. But I do not like this solution because it could shorten the life of the drive.

Does anyone have any suggestions or input here?

**Comment by Recoar** [comment]
I was going to buy those to shuck(pull hdd out of enclosure) and then use for chia which would solve that problem. Would also void warranty on the hdd doing this. as for continuing to use the enclosure I have no idea.

**Comment by chivesz** [comment, marked as answer]
I answered my own question here. Windows looks to be shutting this off to save power. Un checked this box and seems to not go to sleep any more.

![image](https://user-images.githubusercontent.com/34375681/116429902-542d4f80-a814-11eb-9ece-1b7128062dca.png)

**Comment by Recoar** [comment]
Oh right on thanks! I won't have to shuck em now.

________________________________
